common/src/main/java/com/wynntils/screens/guides/gear/WynntilsItemGuideScreen.java

```diff
@@ -95,27 +91,7 @@ public void doRender(PoseStack poseStack, int mouseX, int mouseY, float partialT
 
     private void renderTooltip(PoseStack poseStack, int mouseX, int mouseY) {
         if (hovered instanceof GuideGearItemStackButton guideGearItemStackButton) {
-            GuideGearItemStack itemStack = guideGearItemStackButton.getItemStack();
-
-            List<Component> tooltipLines = itemStack.getTooltipLines(McUtils.player(), TooltipFlag.NORMAL);
-            tooltipLines.add(Component.empty());
-            if (Models.Favorites.isFavorite(itemStack)) {
-                tooltipLines.add(Component.translatable("screens.wynntils.wynntilsGuides.itemGuide.unfavorite")
-                        .withStyle(ChatFormatting.YELLOW));
-            } else {
-                tooltipLines.add(Component.translatable("screens.wynntils.wynntilsGuides.itemGuide.favorite")
-                        .withStyle(ChatFormatting.GREEN));
-            }
-            tooltipLines.add(Component.translatable("screens.wynntils.wynntilsGuides.itemGuide.open")
-                    .withStyle(ChatFormatting.RED));
-            RenderUtils.drawTooltipAt(
-                    poseStack,
-                    mouseX,
-                    mouseY,
-                    0,
-                    tooltipLines,
-                    FontRenderer.getInstance().getFont(),
-                    true);
+            this.renderTooltip(poseStack, guideGearItemStackButton.getItemStack(), mouseX, mouseY);
```

Comment:
Is Qodana valid here?

Reply:
It looks really scary, yes. But I can't see anything but that Qodana is just flat out wrong:
```
	public void renderTooltip(PoseStack poseStack, ItemStack itemStack, int mouseX, int mouseY) {
```
is what we are calling.